api_methods.py

```diff
@@ -0,0 +1,132 @@
+"""Python auto.ria.com API.
+
+Python implementation of API intended for calulating
+average used car prices that are sold on http://auto.ria.com
+"""
+
+
+import requests
+import json
+from fnmatch import fnmatch
+
+
+def jsonify_response(url):
```

Comment:
I would call it `make_request`, as the main feature of this function is to do the http request, not the json conversion.